plugins/orchestrator-backend/src/service/DataInputSchemaService.ts

```diff
@@ -1192,4 +1195,54 @@ export class DataInputSchemaService {
 
     return inputVariableSet;
   }
+
+  public parseComposition(
```

Comment:
I'm managing to create the Stepper form without this parsing
I use the RJSF form as is and pass the schema to it
I override the ObjectFieldTemplate and in it I put the stepper only if the current schema has $defs

Reply:
Nice! In this case, I will undo these changes. I thought it would be useful for you but glad to know that you're progressing.